source/Date.h

```diff
@@ -2,25 +2,37 @@
 #ifndef DATE_H
 #define DATE_H
 
-#include <ctime>
+#include <stdint.h>
 
 class Date
 {
-    
 private:
-    int DAY;
-    int MONTH;
-    int YEAR;
+    uint8_t DAY;
+    uint8_t MONTH;
+    uint8_t YEAR;
     
 public:
-    Date()= default;
-    Date(const int& D, const int& M, const int& Y);
-    now();
+    Date() = default;
+    Date(uint8_t D, uint8_t M, uint8_t Y);
+    
+    /* Getters */
+    uint8_t getDAY() const;
+    uint8_t getMONTH() const;
+    uint8_t getYEAR() const;
     
-    /* Getters*/
-    Date getDate() const;
+    /* Setters */
+    void setDAY(uint8_t D);
+    void setMONTH(uint8_t M);
+    void setYEAR(uint8_t Y);
```

Comment:
These are covered by the setDate function, so they must be removed.

Reply:
Done.